feature/platform/integrated_circuit/otg_tests/utilization_test/utilization_test.go

```diff
@@ -105,36 +107,12 @@ func TestResourceUtilization(t *testing.T) {
 	otgV6Peer, otgPort1, otgConfig := configureOTG(t, otg)
 
 	verifyBgpTelemetry(t, dut)
-
-	val, ok := gnmi.Watch(t, dut, gnmi.OC().System().Utilization().Resource(fibResource[dut.Vendor()]).ActiveComponentList().State(), time.Minute, func(v *ygnmi.Value[[]string]) bool {
-		cs, present := v.Val()
-		return present && len(cs) > 0
-	}).Await(t)
-	if !ok {
-		switch {
-		case deviations.MissingHardwareResourceTelemetryBeforeConfig(dut):
-			t.Log("FIB resource is not active in any available components")
-		default:
-			t.Fatalf("FIB resource is not active in any available components")
-		}
-	}
-	comps, _ := val.Val()
-
 	gnmi.Replace(t, dut, gnmi.OC().System().Utilization().Resource(fibResource[dut.Vendor()]).Config(), &oc.System_Utilization_Resource{
 		Name:                    ygot.String(fibResource[dut.Vendor()]),
 		UsedThresholdUpper:      ygot.Uint8(usedThresholdUpper),
 		UsedThresholdUpperClear: ygot.Uint8(usedThresholdUpperClear),
 	})
-
-	val, ok = gnmi.Watch(t, dut, gnmi.OC().System().Utilization().Resource(fibResource[dut.Vendor()]).ActiveComponentList().State(), time.Minute, func(v *ygnmi.Value[[]string]) bool {
-		cs, present := v.Val()
-		return present && len(cs) > 0
-	}).Await(t)
-	if !ok {
-		t.Fatalf("FIB resource is not active in any available components")
-	}
-	comps, _ = val.Val()
-
+	comps := components.FindActiveComponentsByType(t, dut, oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_INTEGRATED_CIRCUIT)
 	beforeUtzs := componentUtilizations(t, dut, comps)
 	if len(beforeUtzs) != len(comps) {
 		t.Fatalf("Couldn't retrieve Utilization information for all Components in active-component-list")
```

Comment:
Remove "active-component-list" from error message.

Same for line 125

Reply:
Updated , thanks !